index.js

```diff
@@ -6,11 +6,8 @@ module.exports = {
   SpeechToText: {
     speechToTextEmitter: new NativeEventEmitter(RNSpeechToText),
 
-    initialize: function(apiKey) {
```

Comment:
I liked more the idea of having an initialize method with the apiKey, why we are removing it?

Reply:
Ok, I see, I thought we were passing the apiKey in the SpeechToTextSession, but is a bearerToken